tests/test_butler.py

```diff
@@ -649,6 +656,12 @@ def tearDown(self):
         bucket = s3.Bucket(self.bucketName)
         bucket.delete()
 
+        # unset any potentially set dummy credentials
+        keys = ["AWS_ACCESS_KEY_ID", "AWS_SECRET_ACCESS_KEY"]
```

Comment:
No need for this to be a list. Use a tuple.

Reply:
For my personal future reference, why does it make a difference here? I understand tuples are immutable, but this is digested immediately.

Reply:
I think it's always better to be explicit about mutability. You are providing a fixed list of environment variables that will never change so use a tuple. It's no extra typing to do so.

Reply:
This was changed yesterday already, but is also mute point with the newest commit. Will keep that in mind for the future however.